### PR TITLE
Bump to version 0.2.1

### PR DESCRIPTION
**Pull request by blaix**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,8 @@
+0.2.1 (2016-03-30)
```
Comment (adrianmoisey) [inline CHANGELOG.rst]: Not that it really matters, but today is the 31st

**Comment by adrianmoisey** [comment]
:+1:

**Comment by RayeN** [comment]
:+1:

**Pull request merged**